lua/harpoon/buffer.lua

```diff
@@ -86,16 +86,17 @@ function M.setup_autocmds_and_keymaps(bufnr)
         )
     end
     --]]
-    vim.cmd(
-        string.format(
-            "autocmd BufModifiedSet <buffer=%s> set nomodified",
-            bufnr
-        )
-    )
+    vim.api.nvim_create_autocmd("BufModifiedSet", {
+        buffer = bufnr,
+        group = HarpoonGroup,
+        callback = function()
```

Comment:
No real reason for this just a refactor to match the rest of the autocmds here.

Reply:
good